### DISCUSSION TITLE
Option to hide requested items

### DISCUSSION THREAD

**Discussion by tdeverx** [sct/overseerr#2434] [Feature Requests]
Would be super useful to clean up the homepage by adding an option to hide all media that’s already been requested. They would ofc still show when you search for that item, but at least on the homepage, have them hidden.

**Comment by TheCatLady** [comment]
There is an existing similar option to hide available media: https://docs.overseerr.dev/using-overseerr/settings#hide-available-media

**Reply by tdeverx** [reply]
Oh, I know about this feature but I my request was to hide requests, not available media. Unless that feature also hides upcoming shows that technically aren’t “available”? I actually didn’t check haha

**Comment by PeterKse** [comment]
Agree with OP, this is a feature I'd like - when clicking on Movies or Series, I get a lot of already stuff cluttering the view that's already in the cue to get downloaded once it's released. 

In fact, Overseer could probably make a sub category link Movies and Series filtering everything that's not on the disk but is pending download, so users can get a quick overview.
